recipes/grwham/build.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 mkdir -p $PREFIX/bin
 cd "wham_${PKG_VERSION}"
+mkdir share
+curl -o share/doc.pdf http://membrane.urmc.rochester.edu/sites/default/files/wham/doc.pdf
```

Comment:
```suggestion
mkdir $PREFIX/share
curl -o $PREFIX/share/doc.pdf http://membrane.urmc.rochester.edu/sites/default/files/wham/doc.pdf
```

Reply:
This should be unique enough to not stomp on others.

Reply:
@bmabey The change has been updated in 334bfc5